website/src/components/Chat/ChatSection.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  CircularProgress,
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  Flex,
+  FormControl,
+  FormLabel,
+  IconButton,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  Portal,
+  Select,
+  Slider,
+  SliderFilledTrack,
+  SliderThumb,
+  SliderTrack,
+  Stack,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { Settings } from "lucide-react";
+import { useTranslation } from "next-i18next";
+import { useCallback, useState } from "react";
+import { Controller, useForm, UseFormReturn } from "react-hook-form";
+import { get } from "src/lib/api";
+import { API_ROUTES } from "src/lib/routes";
+import { ModelInfo, WorkParametersInput } from "src/types/Chat";
+import useSWRImmutable from "swr/immutable";
+import { StrictOmit } from "ts-essentials";
+
+import { ChatConversation } from "./ChatConversation";
+
+type UseChatConfigForm = UseFormReturn<WorkParametersInput>;
+
+export const ChatSection = ({ chatId }: { chatId: string }) => {
+  const { control, getValues, register, setValue } = useForm<WorkParametersInput>({
+    defaultValues: {
+      temperature: 0.7,
+      top_p: 0.5,
+      top_k: 1.2,
+      repetition_penalty: 1.2,
+      max_new_tokens: 256,
```

Comment:
Here are the default params. I just fill them randomly, please suggest

Reply:
I think the idea of #2107 was to make a request to `INFERENCE/config/models` which will give a pre-defined set of configurations that you could send with the requests. I hope I interpreted this correctly.

Reply:
> I think the idea of #2107 was to make a request to `INFERENCE/config/models` which will give a pre-defined set of configurations that you could send with the requests. I hope I interpreted this correctly.

Let me ask Yannic about this. I'm not sure what behavior he wants.